sample_project/Assets/SampleViewer/Samples/OverviewMap/Scripts/LocationMarker.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright 2024 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+
+using Esri.ArcGISMapsSDK.Components;
+using Esri.ArcGISMapsSDK.Samples.Components;
+using Esri.ArcGISMapsSDK.Utils.GeoCoord;
+using Esri.GameEngine.Geometry;
+using UnityEngine;
+
+public class LocationMarker : MonoBehaviour
+{
+    [SerializeField] private ArcGISCameraComponent cameraComponent;
+
+    private ArcGISCameraControllerComponent cameraController;
+    private ArcGISLocationComponent cameraLocationComponent;
+    private ArcGISLocationComponent locationComponent;
+    private float northOffset = 225.0f;
+
+    private void Awake()
+    {
+        cameraController = FindFirstObjectByType<ArcGISCameraControllerComponent>();
+        cameraLocationComponent = cameraController.GetComponent<ArcGISLocationComponent>();
+        locationComponent = GetComponent<ArcGISLocationComponent>();
+    }
+
+    private void Update()
+    {
+        cameraComponent.GetComponent<ArcGISLocationComponent>().Position = new ArcGISPoint(
+            cameraLocationComponent.Position.X,
+            cameraLocationComponent.Position.Y,
+            cameraComponent.GetComponent<ArcGISLocationComponent>().Position.Z, ArcGISSpatialReference.WGS84());
```

Comment:
We can make this compatible with any reference by projecting the point. No need to force everything to WGS84. Makes it more difficult for a user to use if theyre already using different SRs.